orchestrator/careplanservice/service.go

```diff
@@ -351,17 +356,31 @@ func (s *Service) handleGet(httpRequest *http.Request, httpResponse http.Respons
 func (s *Service) handleSearch(httpRequest *http.Request, httpResponse http.ResponseWriter, resourceType, operationName string) {
 	headers := new(fhirclient.Headers)
 
+	// Ensure the Content-Type header is set correctly
```

Comment:
@JorisHeadease @reinkrul  I added this to deal with a problem of receiving duplicate content-type headers:
`{"level":"warn","time":"2024-12-10T13:22:54Z","message":"Content-Type: application/x-www-form-urlencoded, application/x-www-form-urlencoded"}`

This feels like a bit of a band-aid, do either of you know how we can rather fix this at the source and ensure the header is only set once?

Reply:
Probably better to fix it at the source, did you find what caused the double header value?

Reply:
I agree. Haven't found a cause yet but still investigating. I never ran into this while doing backend E2E tests so I think it is happening somewhere betweenFE and middleware, I will check it out with @JorisHeadease and see what we can find

Reply:
Fixed: https://github.com/SanteonNL/go-fhir-client/pull/4